docs/source/getting_started/benchmarks.rst

```diff
@@ -47,98 +47,75 @@ self-supervised models from scratch on the training data. At the end of every
 epoch we embed all training images and use the features for a kNN classifier 
 with k=200 on the test set. The reported kNN test accuracy is the max accuracy
 over all epochs the model reached.
-All experiments use the same augmentations (SimCLR augmentations without gaussian
-blur) and the same ResNet-18 backbone.
+All experiments use the same ResNet-18 backbone and we disable the gaussian blur
+augmentation due to the small image sizes.
 
 .. note:: The ResNet-18 backbone in this benchmark is slightly different from 
           the torchvision variant as it starts with a 3x3 convolution and has no
           stride and no `MaxPool2d`. This is a typical variation used for cifar10
           benchmarks of SSL methods.
 
 
-The current benchmark contains the following models:
-
-- MoCo (with symmetric loss and memory bank with 4096 entries)
-- SimCLR
-- SimSiam (with 2 MLP layers)
-- BarlowTwins
-- BYOL
-
 .. csv-table:: Cifar10 benchmark results 
-   :header: "Model", "Epochs", "Batch Size", "kNN test Accuracy", "Peak GPU consumption"
-   :widths: 20, 20, 20, 20, 20
-
-   "MoCo", 200, 128, 0.83, "2.1 GBytes"
-   "SimCLR", 200, 128, 0.78, "2.0 GBytes"
-   "SimSiam", 200, 128, 0.73, "3.0 GBytes"
-   "BarlowTwins", 200, 128, 0.84, "2.1 GBytes"                                                                                               
-   "BYOL", 200, 128, 0.85, "2.2 GBytes"
-   "MoCo", 200, 512, 0.85, "7.4 GBytes"
-   "SimCLR", 200, 512, 0.83, "7.8 GBytes"
-   "SimSiam", 200, 512, 0.81, "7.0 GBytes"
-   "BarlowTwins", 200, 512, 0.78, "7.1 GBytes"
-   "BYOL", 200, 128, 0.84, "7.5 GBytes"
-   "MoCo", 800, 128, 0.89, "2.1 GBytes"
-   "SimCLR", 800, 128, 0.87, "1.9 GBytes"
-   "SimSiam", 800, 128, 0.80, "2.0 GBytes"
-   "MoCo", 800, 512, 0.90, "7.2 GBytes"
-   "SimCLR", 800, 512, 0.89, "7.7 GBytes"
-   "SimSiam", 800, 512, 0.91, "6.9 GBytes"
-
-
-You can reproduce the benchmarks using the following script:
-:download:`cifar10_benchmark.py <benchmarks/cifar10_benchmark.py>` 
-
-.. figure:: images/bench_knn-cifar10-200epochs.png
-    :align: center
-    :alt: kNN accuracy on test set for training the models for 200 epochs
-
-    Plot with the kNN accuracy on test set for training the models for 200 epochs. The shorter
-    runs (fewer steps) are the ones with batch size of 512. Green: MoCo, Gray: SimCLR, 
-    Orange: SimSiam, Red: MoCo, Light Blue: SimCLR, Pink: SimSiam.
+  :header: "Model", "Epochs", "Batch Size", "kNN test Accuracy", "Runtime", "Peak GPU consumption"
```

Comment:
For me the additional column cuts off the table on the right, see: 
![grafik](https://user-images.githubusercontent.com/65946090/154990039-98fffd7f-1aea-457d-b56e-51dd65526331.png)

Reply:
Made the column names a bit smaller :)